e2e/dashboards-suite/dashboard-export-image.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { e2e } from '../utils';
+
+describe('Dashboard export image', () => {
+  beforeEach(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+  });
+
+  const openDashboard = () => {
+    cy.intercept({
+      pathname: '/api/ds/query',
+    }).as('query');
+    e2e.flows.openDashboard({
+      uid: 'ZqZnVvFZz',
+    });
+    cy.wait('@query');
+  };
+
+  it('should show renderer not available message when renderer is not installed', () => {
+    openDashboard();
+
+    // Open export menu
+    e2e.pages.Dashboard.DashNav.NewExportButton.arrowMenu().click();
+    e2e.pages.Dashboard.DashNav.NewExportButton.Menu.exportAsImage().click();
+
+    // Verify renderer not available message
+    e2e.components.ExportImage.rendererAlert.container().should('be.visible');
+    e2e.components.ExportImage.rendererAlert.title().should('be.visible');
+    e2e.components.ExportImage.rendererAlert.description().should('be.visible');
+    e2e.components.ExportImage.buttons.generate().should('be.disabled');
+  });
+
+  // TODO: This is a temporary skip until we have a way to mock the renderer plugin
+  it.skip('should generate and download dashboard image when renderer is available', () => {
```

Comment:
is anyone aware of an easy way to mock the image renderer so that Grafana thinks it is available for e2e tests?

Reply:
I would have done the same as you did here and update `grafanaBootData`, I wonder why it isn't working. But I wonder what you want to test here, I think almost everything here should be covered by component testing and the value of the E2E tests would be to test the whole workflow - including the actual image rendering (this is setup in Enterprise but would require some work to make them work in OSS).

Reply:
that's a fair point - will opt to remove these e2e tests for the time being in favor of current unit test coverage 👍